src/com/jwetherell/algorithms/numbers/Integers.java

```diff
@@ -6,6 +6,14 @@
 
 public class Integers {
 
+    // moved constants to the top of the class definition for better visibility
```

Comment:
This change helps clarify what the constants are for the class since they were in the middle of the class definition earlier.